# rxjs-throwerror-inner-observable

[Edit on StackBlitz ⚡️](https://stackblitz.com/edit/rxjs-throwerror-inner-observable)